pkg/account/deployer/deployer.go

```diff
@@ -22,15 +22,28 @@ import (
 	accountmanagement "github.com/dynatrace/dynatrace-configuration-as-code-core/gen/account_management"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/internal/log"
 	"github.com/dynatrace/dynatrace-configuration-as-code/v2/pkg/account"
+	"strings"
 )
 
 type (
 	localId    = string // local (monaco related) identifier
 	envName    = string // dt environment name
 	remoteId   = string // dt entity identifier
 	idLookupFn func(id localId) remoteId
+
+	permissions []accountmanagement.PermissionsDto
 )
 
+func (p permissions) String() string {
+	sb := strings.Builder{}
+	sb.WriteString("[")
+	for _, e := range p {
+		fmt.Fprintf(&sb, "{%s %s %s}", e.PermissionName, e.ScopeType, e.Scope)
```

Comment:
Do you have a sample of how this prints? I wonder if just printing the three strings is clear enough or if we should add more details (or even print less to give users the main info they need)

Reply:
```
[{permission-name account abcde-fghijk}{tenant-viewer tenant abd1392}]
```
so more or less like a `%v` on the regular struct but without CreatedAt and other fields.....
And I've removed e.Scope for not leaking account UUID (see updated PR)